BaseApi/V1/Controllers/DayCarePackageController.cs

```diff
@@ -14,15 +16,21 @@ namespace BaseApi.V1.Controllers
     public class DayCarePackageController : ControllerBase
     {
         private readonly ICreateDayCarePackageUseCase _createdDayCarePackageUseCase;
+        private readonly IGetDayCarePackageUseCase _getDayCarePackageUseCase;
 
-        public DayCarePackageController(ICreateDayCarePackageUseCase createdDayCarePackageUseCase)
+        public DayCarePackageController(ICreateDayCarePackageUseCase createdDayCarePackageUseCase, IGetDayCarePackageUseCase getDayCarePackageUseCase)
         {
             _createdDayCarePackageUseCase = createdDayCarePackageUseCase;
+            _getDayCarePackageUseCase = getDayCarePackageUseCase;
         }
-
+        /// <summary>
+        /// Create day care package
+        /// </summary>
+        /// <param name="dayCarePackageForCreation"></param>
```

Comment:
> You may want to fill the comment space for each parameter, as this will provide a description of the parameter to auto-docs

Good point. Will fix that